Journalator_Display/Source/Display/InvestONator/PortfolioDisplay.lua

```
@@ -0,0 +1,423 @@
+  self.NameEditBox = nameEditBox
+  self.InvestmentEditBox = investmentEditBox
+end
+
+---Refresh the portfolio list display
+-- Rebuilds the portfolio list UI from current portfolio data.
+-- Hides and detaches any existing portfolio frames, creates and positions a frame for each portfolio, and updates the content container height.
+-- Ensures a "Create" button exists to open the portfolio creation dialog.
+function JournalatorInvestONatorPortfolioDisplayMixin:RefreshPortfolioList()
+  -- Clear existing frames
+  self:ReleaseAllPortfolioFrames()
+  
```

Luacheck (1.2.0)

[warning] line contains only whitespace

(W611)